cmd/notation/list.go

```diff
@@ -55,9 +59,26 @@ func runList(command *cobra.Command, opts *listOpts) error {
 	}
 
 	// write out
-	for _, sigManifest := range sigManifests {
-		fmt.Println(sigManifest.Blob.Digest)
+	return output(manifestDesc.Digest, sigManifests, reference)
+}
+
+func output(digest digest.Digest, sigManifests []notationRegistry.SignatureManifest, reference string) error {
```

Comment:
It is better to choose a good name like `printSignatureManifestList` as `output` is too generic and could be easily misused.

Reply:
Renamed to `printSignatureManifestList`